test/test_pure_market_making_v2.py

```diff
@@ -516,3 +549,73 @@ def test_balance_for_multiple_staggered_orders(self):
         self.clock.backtest_til(end_ts)
         self.assertEqual(5, len(self.multi_order_staggered_strategy.active_bids))
         self.assertEqual(5, len(self.multi_order_staggered_strategy.active_asks))
+
+    def test_inventory_skew_single_order_strategy(self):
+        self.clock.remove_iterator(self.strategy)
+        self.clock.add_iterator(self.inventory_skew_single_order_strategy)
+        self.clock.backtest_til(self.start_timestamp + self.clock_tick_size)
+        self.assertEqual(1, len(self.inventory_skew_single_order_strategy.active_bids))
+        self.assertEqual(1, len(self.inventory_skew_single_order_strategy.active_asks))
+        first_bid_order: LimitOrder = self.inventory_skew_single_order_strategy.active_bids[0][1]
+        first_ask_order: LimitOrder = self.inventory_skew_single_order_strategy.active_asks[0][1]
+        self.assertEqual(Decimal("99"), first_bid_order.price)
+        self.assertEqual(Decimal("101"), first_ask_order.price)
+        self.assertEqual(Decimal("0.181818"), first_bid_order.quantity)
+        self.assertEqual(Decimal("1.81818"), first_ask_order.quantity)
+
+        self.simulate_maker_market_trade(True, 5.0)
+        self.assertEqual(1, len(self.inventory_skew_single_order_strategy.active_bids))
+        self.assertEqual(0, len(self.inventory_skew_single_order_strategy.active_asks))
+
+        self.clock.backtest_til(self.start_timestamp + 2 * self.clock_tick_size + 1)
+        self.assertEqual(1, len(self.maker_order_fill_logger.event_log))
+
+        maker_fill: OrderFilledEvent = self.maker_order_fill_logger.event_log[0]
+        self.assertEqual(TradeType.SELL, maker_fill.trade_type)
+        self.assertAlmostEqual(101, maker_fill.price)
+        self.assertAlmostEqual(Decimal("1.81818"), Decimal(str(maker_fill.amount)), places=4)
+
+        self.strategy.cancel_order(self.market_info, first_bid_order.client_order_id)
+        self.clock.backtest_til(self.start_timestamp + 2 * self.clock_tick_size + 1)
+        self.assertEqual(0, len(self.strategy.active_bids))
+        self.assertEqual(0, len(self.strategy.active_asks))
```

Comment:
The unit tests here are not adequate... the important thing to test is whether the orders created have the scaled size in an unbalanced situation (after that 5.0 order fill). But instead you're just testing for the order fill events here.